scene/animate-images-with-image-overlay/src/main/resources/animate_images_with_image_overlay.fxml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<?import javafx.scene.layout.*?>
+
+<?import com.esri.arcgisruntime.mapping.view.SceneView?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.paint.Color?>
+<?import javafx.scene.control.Label?>
+<StackPane fx:controller="com.esri.samples.animate_images_with_image_overlay.AnimateImagesWithImageOverlayController"
+           xmlns:fx="http://javafx.com/fxml" stylesheets="/style.css">
+    <SceneView fx:id="sceneView"/>
+    <VBox StackPane.alignment="BOTTOM_CENTER" maxWidth="-Infinity" maxHeight="50" styleClass="panel-region" alignment="TOP_CENTER">
+    <StackPane.margin>
+        <Insets bottom="20"/>
+    </StackPane.margin>
+    <padding>
+        <Insets topRightBottomLeft="20"/>
+    </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.3"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
```

Comment:
Every time I see this in FXML files I cringe a bit 😞 
I've recently used CSS to get rid of some of this stuff and hide it in styles.css:
https://github.com/Esri/arcgis-runtime-samples-java/pull/498/files

Perhaps we can make that a regular thing with new samples? 


The trick is to add `styleClass="stack-pane"` to the StackPane, and then you can target individual FXML elements with rules, e.g. for a VBox:
```
.stack-pane VBox {
  -fx-text-fill: white;
    -fx-background-color: rgba(0, 0, 0, 0.3);
    -fx-padding: 10;
    -fx-max-width: 200;
    -fx-max-height: 120;
    -fx-spacing: 10;
}
``` 

It'll also allow you to hide away a lot of the `max width` etc. definitions.